src/patternfly/demos/Table/examples/Table.md

```diff
@@ -283,7 +283,7 @@ section: demos
     {{#> page-main-section page-main-section--modifier="pf-m-light"}}
       {{> table-main-section-content}}
     {{/page-main-section}}
-    {{#> page-main-section page-main-section--modifier="pf-m-no-padding-mobile"}}
+    {{#> page-main-section page-main-section--modifier="pf-m-no-padding-on-md"}}
```

Comment:
Anywhere we were using `pf-m-no-padding-mobile` now needs to be `pf-m-no-padding pf-m-padding-on-md`. Otherwise you're adding it on mobile (since the page section has padding by default), and removing it on desktop with `pf-m-no-padding-on-md`. Here's an example from the build in this PR:

<img width="1201" alt="Screen Shot 2020-03-24 at 8 11 00 PM" src="https://user-images.githubusercontent.com/35148959/77491410-b181f080-6e0b-11ea-9a6b-9c7496ef1505.png">
<img width="666" alt="Screen Shot 2020-03-24 at 8 10 56 PM" src="https://user-images.githubusercontent.com/35148959/77491413-b21a8700-6e0b-11ea-8bfd-79e408ee2ac7.png">

Reply:
Thanks forgot to update those :(